templates/terraform/processing-project/buckets.tf

```
@@ -95,3 +95,20 @@ resource google_storage_bucket_iam_member staging_account_iam_reader {
   role = "roles/storage.objectViewer"
   member = "serviceAccount:${var.jade_repo_email}"
 }
+
+# Bucket for long term Argo logs storage, currently want no "delete after N days" rule.
+resource google_storage_bucket argo_logs_storage {
+  provider = google.target
+  name = "${var.project_name}-argo-logs-storage"
```

Could we do this instead?
```suggestion
  name = "${var.project_name}-argo-archive"
```
Argo technically supports archiving files other than logs. Idk if we'll ever make use of that feature, but if we do then the files will end up in this bucket by default.